core/src/docs/rfcs/3243_list_prefix.md

```diff
@@ -93,7 +93,7 @@ While create dir on a path:
 
 # Reference-level explanation
 
-For POSIX-alike services, we will:
+For POSIX-like services, we will:
```

Comment:
Hi, I personnally believe the usage here is correct.

The answer from ChatGPT about "what's the different between xxx-like and xxx-alike":


> The suffixes "-like" and "-alike" both serve to compare or denote similarity, but they are used in slightly different contexts and can carry different connotations:
>
> 1. **-like**: This suffix is used to indicate that something resembles or has characteristics similar to the thing mentioned. It is often used to describe inherent qualities or physical resemblances. For example:
>    - "Childlike" suggests possessing qualities or characteristics of a child.
>    - "Lifelike" means resembling real life or appearing very much alive.
>
> 2. **-alike**: This suffix is commonly used to emphasize a high degree of similarity between things or people, often in behavior or manner, rather than just appearance. It can also be used to indicate that the similarity applies to a group rather than an individual characteristic. For example:
>    - "Look-alike" refers to someone who looks very similar to someone else, usually used in the context of two people who are not related.
>    - "Think-alike" implies having similar thoughts or opinions.
>
> In summary, while both suffixes denote similarity, "-like" is more about qualities or appearance, and "-alike" often emphasizes similarity in actions, behaviors, or among groups.

Reply:
Hi, IMHO what GPT said here might be incorrect. According to Merriam-Webster Dictionary ([like](https://www.merriam-webster.com/dictionary/like#dictionary-entry-9), [alike](https://www.merriam-webster.com/dictionary/alike)), Cambridge Dictionary ([like](https://dictionary.cambridge.org/dictionary/english/like), [alike](https://dictionary.cambridge.org/dictionary/english/alike)) and Oxford Advanced Learner's Dictionary ([like](https://www.oxfordlearnersdictionaries.com/definition/english/like_2), [alike](https://www.oxfordlearnersdictionaries.com/definition/english/alike_1)), the suffix combining form is only applicable to "-like" but not "-alike". "Look-alike" is a whole word but not a combination of "look" and "-alike". Also, there's much less search result on Google for keyword "posix-alike" than "posix-like".

Reply:
Thanks for pointing out.